.gitignore

```diff
@@ -29,9 +29,6 @@ pids
 # Directory for instrumented libs generated by jscoverage/JSCover
 lib-cov
 
-# Coverage directory used by tools like istanbul
-coverage
```

Comment:
Removed this as it excluded the `coverage` directory in the Foundry code and we don't use JS code coverage